exporter/clickhouselogsexporter/exporter.go

```diff
@@ -164,10 +165,20 @@ func (e *clickhouseLogsExporter) pushLogsData(ctx context.Context, ld plog.Logs)
 			}
 		}
 	}
+
+	dbWriteStart := time.Now()
 	err = statement.Send()
 	if err != nil {
 		return fmt.Errorf("StatementSend:%w", err)
 	}
+	stats.RecordWithTags(ctx,
+		[]tag.Mutator{
+			tag.Upsert(exporterKey, string(component.DataTypeLogs)),
+			tag.Upsert(tableKey, DISTRIBUTED_LOGS_TABLE),
+		},
+		writeLatencyMillis.M(int64(time.Since(dbWriteStart).Milliseconds())),
+	)
```

Comment:
This should be before `err` return block. Say it errd with timeout you would want to record because that spike indicates something unusual. Not recording that means losing the good data point.

Reply:
Yeah, makes sense for this metric.